### DISCUSSION TITLE
in mysql server is not running any commands

### DISCUSSION THREAD

**Discussion by NARENDRA935** [learndevopsonline/learndevopsonline#1691] [Q&A]
hii, @r-devops ,
in mysql server is not running any commands if execute the command then its showing killed,please sort out this problem as soon as possible
![Uploading Screenshot 2023-12-21 141239.png…]()

**Comment by sreedharm07** [comment]
your screen shot is not able to view
wait until it uploads

**Reply by NARENDRA935** [reply]
issue  is 
solved
